app/models/placement_type.rb

```diff
@@ -1,6 +1,8 @@
 class PlacementType < ApplicationRecord
   belongs_to :casa_org
+
   validates :name, presence: true
+  scope :for_organization, ->(org) { where(casa_org: org).order(:name) }
```

Comment:
I'd prefer separate scopes for ordering, especially if it's not in the scope name!